solidity/contracts/BondedECDSAKeepFactory.sol

```diff
@@ -277,12 +277,12 @@ contract BondedECDSAKeepFactory is IBondedECDSAKeepFactory, CloneFactory {
     /// @notice Sets a new group selection seed value.
     /// @dev The function is expected to be called in a callback by the random
     /// beacon.
-    /// @param _groupSelectionSeed New value of group selection seed.
-    function setGroupSelectionSeed(uint256 _groupSelectionSeed)
+    /// @param _relayEntry Beacon output.
+    function __beaconCallback(uint256 _relayEntry)
```

Comment:
Thoughts on defining an interface on beacon side that would define `function __beaconCallback(uint256 _relayEntry)` function and be implemented by `BondedECDSAKeepFactory` contract?

Reply:
No thoughts on this at the moment, opened an issue on `keep-core` side to address it next week: https://github.com/keep-network/keep-core/issues/1537